fermenter-mda/src/test/java/org/technologybrewery/fermenter/mda/element/CommonSteps.java

```diff
@@ -4,51 +4,67 @@
 
 import org.aeonbits.owner.KrauseningConfigFactory;
 import org.apache.commons.io.FileUtils;
+import org.technologybrewery.fermenter.mda.generator.GenerationException;
 import org.technologybrewery.fermenter.mda.metamodel.MetamodelConfig;
 import org.technologybrewery.fermenter.mda.util.MessageTracker;
 
-import cucumber.api.java.After;
-import cucumber.api.java.Before;
-
-public class CommonSteps {
+public final class CommonSteps {
```

Comment:
A: add a comment to explain why this is here and not using `@Before` and `@After` anymore.